src/torrent_summary.rs

```diff
@@ -6,6 +6,27 @@ pub(crate) struct TorrentSummary {
   size: Bytes,
 }
 
+#[derive(Serialize)]
+pub(crate) struct TorrentSummaryData {
```

Comment:
I think this should be named to indicate that it's the JSON summary, since otherwise it's confusing to know what it's for. Perhaps `TorrentSummaryJson`?